config/routes.rb

```diff
@@ -1,4 +1,12 @@
 Rails.application.routes.draw do
+  namespace :api, defaults: { format: :json } do
+    resources :users, only: [:index, :show] do
+      resources :posts, only: [:index] do
+        resources :comments, only: [:index, :create]
+      end
+    end
+  end
```

Comment:
- [ ] Kindly make sure all your API endpoints receives correct data and responds in a JSON format as well. Right now, after testing your endpoints with [postman](https://web.postman.co/), I noticed that no one of them is working.